scrapy_poet/downloadermiddlewares.py

```diff
@@ -107,6 +108,16 @@ def process_response(
         and an injectable attribute,
         the user-defined ``cb_kwargs`` takes precedence.
         """
+        if request.callback is None and spider.parse.__annotations__:
```

Comment:
Would it be possible to only warn if annotated types are going to be handled by scrapy-poet? For example, one can annotate Response as scrapy.Response, or there might be another argument, passed through cb_kwargs.

Reply:
Good call! Updated in https://github.com/scrapinghub/scrapy-poet/pull/106/commits/b03296b72297a46825fc67381992162a74f2e12c.